spec/amp-video-interface.md

```diff
@@ -16,22 +16,33 @@ these players implement.
 
 ### For 3rd party services
 
--   [amp-3q-player](https://amp.dev/documentation/components/amp-3q-player)
--   [amp-brid-player](https://amp.dev/documentation/components/amp-brid-player)
--   [amp-brightcove](https://amp.dev/documentation/components/amp-brightcove)
--   [amp-dailymotion](https://amp.dev/documentation/components/amp-dailymotion)
--   [amp-delight-player](https://amp.dev/documentation/components/amp-delight-player)
--   [amp-gfycat](https://amp.dev/documentation/components/amp-gfycat)
--   [amp-ima-video](https://amp.dev/documentation/components/amp-ima-video)
--   [amp-minute-media-player](https://amp.dev/documentation/components/amp-minute-media-player)
--   [amp-mowplayer](https://amp.dev/documentation/components/amp-mowplayer)
--   [amp-nexxtv-player](https://amp.dev/documentation/components/amp-nexxtv-player)
--   [amp-ooyala-player](https://amp.dev/documentation/components/amp-ooyala-player)
--   [amp-powr-player](https://amp.dev/documentation/components/amp-powr-player)
--   [amp-vimeo](https://amp.dev/documentation/components/amp-vimeo)
--   [amp-viqeo-player](https://amp.dev/documentation/components/amp-viqeo-player)
--   [amp-wistia-player](https://amp.dev/documentation/components/amp-wistia-player)
--   [amp-youtube](https://amp.dev/documentation/components/amp-youtube)
+<!--
+  The following list is generated automatically:
+
+    gulp check-video-interface-list --fix
+
+  These link to Github instead of amp.dev since we couldn't otherwise check
+  existence of links to extensions added in a pull request.
+-->
```

Comment:
For consistency, WDYT about changing the links in the `Generic` section above to point to GitHub as well?

Reply:
Since they get excluded, I think it's better to include a link to amp.dev since it's more featured.